shell/platform/linux/fl_renderer.cc

```diff
@@ -58,6 +58,244 @@ static const gchar* get_egl_error() {
   }
 }
 
+// Converts an EGL decimal value to a string.
+static gchar* egl_decimal_to_string(EGLint value) {
+  return g_strdup_printf("%d", value);
+}
+
+// Converts an EGL hexadecimal value to a string.
+static gchar* egl_hexadecimal_to_string(EGLint value) {
+  return g_strdup_printf("0x%x", value);
+}
+
+// Converts an EGL enumerated value to a string.
+static gchar* egl_enum_to_string(EGLint value) {
+  if (value == EGL_FALSE)
+    return g_strdup("EGL_FALSE");
+  else if (value == EGL_LUMINANCE_BUFFER)
+    return g_strdup("EGL_LUMINANCE_BUFFER");
+  else if (value == EGL_NONE)
+    return g_strdup("EGL_NONE");
+  else if (value == EGL_NON_CONFORMANT_CONFIG)
+    return g_strdup("EGL_NON_CONFORMANT_CONFIG");
+  else if (value == EGL_RGB_BUFFER)
+    return g_strdup("EGL_RGB_BUFFER");
+  else if (value == EGL_SLOW_CONFIG)
+    return g_strdup("EGL_SLOW_CONFIG");
+  else if (value == EGL_TRANSPARENT_RGB)
+    return g_strdup("EGL_TRANSPARENT_RGB");
+  else if (value == EGL_TRUE)
+    return g_strdup("EGL_TRUE");
+  else
+    return nullptr;
+}
+
+// Removes the bit from the bitfield if it was there.
```

Comment:
The comment and the name of this function are dramatically different; this should be called something like `remove_bit_if_present` (and the return value added to the comment).